doc/architecture-decisions/0008-controller-specs.txt

```diff
@@ -0,0 +1,40 @@
+# 1. Testing controllers
+
+Date: 2022-10-18
+
+1.
+"controller tests" have filenames like `spec/controllers/casa_admins_controller_spec.rb` and syntax like
+```
+allow(controller).to receive(:authenticate_user!).and_return(:supervisor)
+log_in_as_admin
+get :index
+expect(response).to be_successful
+```
+2.
+"request tests" have filenames like `spec/requests/contact_types_spec.rb` and syntax like
+```
+sign_in_as_admin
```

Comment:
I would just use:
```suggestion
sign_in admin
```

as the example which just uses the Devise helper to sign in.